docs/network/default_cni_plugin.md

```diff
@@ -68,7 +68,7 @@ The following configs are supported in ovn-kubernetes config file:
 |ovsInit.disableOVSInit           |N        |bool    |false   |Skip configuring OVS bridge "br-ex" in microshift-ovs-init.service|true   |
 |ovsInit.gatewayInterface         |N        |string  |""      |Interface to be added in OVS gateway bridge "br-ex"               |eth0   |
 |ovsInit.externalGatewayInterface |N        |string  |""      |Interface to be added in external OVS gateway bridge "br-ex1"     |eth1   |
-|mtu                              |N        |uint32  |1400    |MTU value to be used for the Pods                                 |1300   |
+|mtu                              |N        |uint32  |1500    |MTU value to be used for the Pods                                 |1300   |
```

Comment:
Do we need to update the description?
Also, the example of 1300, can we change it to something that one would actually set?

Reply:
updated